builder/builder.mk

```diff
@@ -146,7 +146,7 @@ INGRESS_TEST_LOCAL_ADMIN_PORT = 8877
 INGRESS_TEST_MANIF_DIR = $(BUILDER_HOME)/../manifests/emissary/
 INGRESS_TEST_MANIFS = emissary-crds.yaml emissary-emissaryns.yaml
 
-# export DOCKER_BUILDKIT := 0
+export DOCKER_BUILDKIT := 1
```

Comment:
Not that I expect `docker build` to stick around too much longer (yay ocibuild), but there should probably be a comment here about us using buildkit syntax extensions, and a link to https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md